package.json

```
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint src",
     "test": "jest src",
-    "build": "rm -rf build && ncc build --minify"
+    "build": "rm -rf build && ncc build index.js -o dist --minify"
```

I also dug a bit into this. It seems like `ncc` has some kind of cache which would result in outdated builds. Even passing `no-cache` to the CLI doesn't make a difference.

But I noticed the above script is wrong: The folder for the built assets isn't called "build", but "dist". It seems like once that is fixed, the script works as expected.

```suggestion
    "build": "rm -rf dist && ncc build"
```